### DISCUSSION TITLE
Native Pico-8 support

### DISCUSSION THREAD

**Discussion by donhilion** [eduardofilo/RG350_adam_image#160] [Ideas & Requests]
It would be nice to be able to use the native pico-8.

**Comment by eduardofilo** [comment]
What do you mean with native?

**Reply by donhilion** [reply]
As I understand it, at the moment the "emulator" Tac08 ist used. I purchased Pico-8 and would like to put the raspberry pie (or linux) files in a folder to use the original Pico-8 console.

But as I looked up the SOC specs, it seemes, that this is not an ARM based chip. I guess, we will have to wait until Pico-8 is compiled for this architecture.

**Reply by eduardofilo** [reply]
Ah, OK, now I understand. In addition to the architecture problem, I could only include the official emulator in the image if it could be freely distributed, and I think it is not.
